backend/src/main/java/gov/cdc/usds/simplereport/service/PersonService.java

```diff
@@ -176,7 +190,7 @@ protected Specification<Person> buildPersonMatchFilter(
    * @param facilityId If null, then it means across accessible facilities in the whole organization
    * @param pageOffset Pagination offset is zero based
    * @param pageSize How many results to return, zero will result in the default page size (large)
-   * @param isArchived Default is false. true will ONLY show deleted users
+   * @param includeArchivedPatients Default is false, true will ONLY show deleted users
```

Comment:
`true will ONLY show deleted users` 
is this still accurate after this change?

Reply:
Yes. I preserved the behavior of this `getPatients` service method since it is used widely in tests and other code and would be a big pain to unwind.

However, this leaves us without a way of getting all patients regardless of status! To that end, I also added an overloaded `getPatients` service method that allows the inclusion of _both_ active and archived users. It is this overloaded method that is called by the `getPatients` resolver method.

Does that make sense? It's a bit of a minefield of backwards compatibility so definitely open to suggestions for improvement.

Reply:
imo this parameter name should stay `isArchived` because `includeArchived` to me implies that we're not necessarily excluding active